config/environments/production.rb

```diff
@@ -30,15 +30,15 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options)
-  config.active_storage.service = :amazon
+  config.active_storage.service = ENV['ACTIVE_STORAGE_SERVICE']&.to_sym || :amazon
 
   # Mount Action Cable outside main process or domain
   # config.action_cable.mount_path = nil
   # config.action_cable.url = 'wss://example.com/cable'
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = false
```

Comment:
In order to access the staging environment via the IP address, I had to disable the `force_ssl` property. If there's a better way to do this, or if we need to setup DNS/SSL I can certainly do that and revert this change.